### ISSUE TITLE
install on Mac not downloading correct package

### ISSUE THREAD

**Issue by aneal** [richin13/asdf-neovim#21]
As of release v0.10.0 the packages have an ARCH in the name and downloading nvim-macos.tar.gz is no longer valid:
``` 
>asdf install neovim latest
==> curl -Lo /var/folders/h6/st58r9s51937smhhcp51p54m0000gp/T//neovim-0.10.0.tar.gz -C - https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-macos.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     75      0 --:--:-- --:--:-- --:--:--    75
tar: Error opening archive: Unrecognized archive format
```
Asset list from the [v0.10.0 release page](url):
```
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-linux64.tar.gz
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-linux64.tar.gz.sha256sum
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-macos-arm64.tar.gz
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-macos-arm64.tar.gz.sha256sum
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-macos-x86_64.tar.gz
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-macos-x86_64.tar.gz.sha256sum
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-win64.msi
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-win64.msi.sha256sum
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-win64.zip
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim-win64.zip.sha256sum
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim.appimage
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim.appimage.sha256sum
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim.appimage.zsync
https://github.com/neovim/neovim/releases/download/v0.10.0/nvim.appimage.zsync.sha256sum
https://github.com/neovim/neovim/archive/refs/tags/v0.10.0.zip
https://github.com/neovim/neovim/archive/refs/tags/v0.10.0.tar.gz
```

**Comment by richin13** [comment]
This should be fixed by changes introduced in #22 by @rm-kierancoker , please update the plugin and try again 👍🏼

**Comment by aneal** [comment]
> This should be fixed by changes introduced in #22 by @rm-kierancoker , please update the plugin and try again 👍🏼

I was able to install and update the plugin from the github repo directly and install neovim v0.10.0 successfully.

**Comment by ansidev** [comment]
I guess we can close this issue.
